qutebrowser/browser/browsertab.py

```diff
@@ -477,6 +477,12 @@ def can_go_back(self):
     def can_go_forward(self):
         raise NotImplementedError
 
+    def itemAt(self, i):
```

Comment:
qutebrowser follows Python's instead of Qt's naming conventions for stuff it wraps. Also, since this isn't needed from outside this class, it should be "private" [by adding a `_` in front](https://docs.python.org/3/tutorial/classes.html#private-variables).

So this should be named `_item_at`, and `goToItem` should be `_go_to_item`.